src/hooks/useScore.ts

```diff
@@ -2,20 +2,25 @@ import { routes } from '@/constants/routes';
 import axios from 'axios';
 import { useQuery } from 'react-query';
 import { ScoreState } from './types';
+import { useAccount } from 'wagmi';
 
 type Props = {
   userAddress: `0x${string}`;
   gameId: string;
 };
 
-export function useScore({ userAddress, gameId }: Props) {
+export function useScore() {
+  const { address: userAddress, isDisconnected } = useAccount();
```

Comment:
why do you need this? I don't see it being used in this file.

Reply:
>why not get these from the caller?
Its already available in the hook, cleaner to me to just keep it local there rather than add more to the caller

Reply:
my thinking is we want to keep it functional when we can in order to keep components / hook as dumb as possible.

If we were to write a test for `useScore` now we have to mock out `useAccount` as well. Before, all we had to do is pass userAddress and gameId. very clear with no side effects

Reply:
Ok I see, will update 🤝